src/QuickLinks/index.test.js

```diff
@@ -56,21 +56,23 @@ describe('ReportManager - QuickLinks', () => {
   });
 
   describe('QuickLinks.Anchor', () => {
-    test('triggers onClickAnchor', async () => {
-      const getSectionElement = jest.fn(() => true), onClickAnchor= jest.fn();
+    test('triggers onClickAnchor and the onClick callback if it is provided', async () => {
+      const getSectionElement = jest.fn(() => true), onClickAnchor= jest.fn(), onClick= jest.fn();
       render(
         <QuickLinksContext.Provider value={{ getSectionElement, onClickAnchor }}>
-          <QuickLinks.Anchor anchorTitle="anchor1" iconComponent={<BulletListIcon />} />
+          <QuickLinks.Anchor anchorTitle="anchor1" iconComponent={<BulletListIcon />} onClick={onClick} />
```

Comment:
These having the same anchor title means they have the same test ID selector, which in the future may lead to weird behavior in this test. Just change one of the titles and you're good to go 👍

Reply:
Sorry @JoshuaVulcan , I'm confused by your comment. There is only one anchor with title `anchor1`, and that is how we are referring to it in the tests. What do you mean with "these having the same anchor title"?